### PR TITLE
[WebBundle] Locale and Currency menu shouldn't be renderd when only 1 exists

### PR DESCRIPTION
**Pull request by pix-art**
It's weird the menu's are shown when there is no option to cycle through. Currency is shown in many places so you don't need a weird sign on the top to tell you the current currency. Same goes with the language.

### CODE REVIEW

**Comment by jverdeyen** [comment]
:+1: makes sense

**Comment by stloyd** [comment]
@pjedrzejewski ping.

File: src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php [hunk 1]
```diff
@@ -181,21 +181,27 @@ public function createMainMenu()
...
+
         $menu = $this->factory->createItem('root', array(
             'childrenAttributes' => array(
                 'class' => 'nav nav-pills'
             )
         ));
 
-        foreach ($this->currencyProvider->getAvailableCurrencies() as $currency) {
-            $code = $currency->getCode();
+        if (count($currencies) == 1) {
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php]: 1 === count($currencies)

File: src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php [hunk 2]
```diff
@@ -181,21 +181,27 @@ public function createMainMenu()
...
+            $menu->setDisplay(false);
+            return $menu;
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php]: You should add an extra line

File: src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php [hunk 3]
```diff
@@ -55,27 +55,29 @@ public function __construct(
...
     {
+        $locales = $this->localeProvider->getAvailableLocales();
         $menu = $this->factory->createItem('root', array(
             'childrenAttributes' => array(
                 'class' => 'nav nav-pills'
             )
         ));
-
-        foreach ($this->localeProvider->getAvailableLocales() as $locale) {
+        if (count($locales) == 1) {
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Menu/LocaleMenuBuilder.php]: idem

**Comment by aRn0D** [comment]
:+1: yep, it make sense !

**Comment by pix-art** [comment]
Fixed typos :+1:

**Comment by pix-art** [comment]
The test fails don't seem to be my fault? Or am i mistaking?

**Comment by stloyd** [comment]
@pix-art Can you rebase your code & squash commits? Thanks!
@pjedrzejewski Ping...

**Comment by pix-art** [comment]
Sorry i merged and forgot this would show here ☺️ I'll sort this tonight or tomorrow

**Comment by pix-art** [comment]
I f*cked stuff up, i'll re-submit a new PR. Sorry for this first time trying this stuff out :)

**Pull request closed without merging**